roofit/roofitcore/src/Initialisation.cxx

```diff
@@ -62,25 +69,31 @@ void loadComputeLibrary() {
   }
 
 #if __GNUC__ > 5 || defined(__clang__)
-  if (__builtin_cpu_supports("avx512f")) {
-    libSuffix = "_AVX512f";
+  //skylake-avx512 support
+  if (__builtin_cpu_supports("avx512cd") && __builtin_cpu_supports("avx512vl") && __builtin_cpu_supports("avx512bw") && __builtin_cpu_supports("avx512dq"))  {
+    libSuffix = "_AVX512";
   }
 #endif
 
-#endif
 
   for (auto&& libName : libNames) {
     libName += libSuffix;
     const auto returnValue = gSystem->Load(libName.c_str());
 
     if (returnValue == -1 || returnValue == -2) {
-      throw std::runtime_error("RooFit was unable to load its computation library " + libName);
+      std::cout << "RooFit was unable to load its computation library " + libName << std::endl;
     }
     // Library should not have been loaded before we tried to do it.
-    if (returnValue == 1) {
+    else if (returnValue == 1) {
       throw std::logic_error("RooFit computation library " + libName + " was loaded before RooFit initialisation began.");
     }
+    else if (gDebug>0) {
+      std::cout << "Library " + libName + " was loaded succesfully" << std::endl;
```

Comment:
Please add also here `"In RooFit:...(): "` to give some context to the experiments looking at logs.